pkg/virtctl/create/instancetype/instancetype_test.go

```diff
@@ -140,44 +168,35 @@ var _ = Describe("create instancetype", func() {
 			Entry("VirtualMachinePreference", true),
 			Entry("VirtualMachineClusterPreference", false),
 		)
+		testDeviceConfig := func(flagName, deviceParam string, validateFunc func(spec *instancetypev1beta1.VirtualMachineInstancetypeSpec)) {
+			DescribeTable(fmt.Sprintf("with defined %s", flagName), func(extraArgs ...string) {
```

Comment:
Can you please keep `DescribeTable` out of the helper function and only put the following into a helper function? The helper function should preferrably be a regular function, not a closure.

```
				args := append([]string{
					setFlag(CPUFlag, "1"),
					setFlag(MemoryFlag, "128Mi"),
					setFlag(flagName, deviceParam),
				}, extraArgs...)
				out, err := runCmd(args...)
				Expect(err).ToNot(HaveOccurred())
```